opengrok-indexer/src/main/java/org/opengrok/indexer/web/SearchHelper.java

```diff
@@ -314,12 +314,14 @@ public SearchHelper prepareExec(SortedSet<String> projects) {
                 // We use MultiReader even for single project. This should
                 // not matter given that MultiReader is just a cheap wrapper
                 // around set of IndexReader objects.
-                reader = RuntimeEnvironment.getInstance().getMultiReader(
-                    projects, searcherList);
+                reader = RuntimeEnvironment.getInstance().getMultiReader(projects, searcherList);
                 if (reader != null) {
                     searcher = new IndexSearcher(reader);
                 } else {
-                    errorMsg = "Failed to initialize search. Check the index.";
+                    errorMsg = "Failed to initialize search. Check the index";
+                    if (projects.size() > 0) {
+                        errorMsg = errorMsg + " for projects: " + String.join(", ", projects);
```

Comment:
Another nitpick, why not use `+=`? I personally like it more.

Reply:
good idea